source/features/remove-actions-tab.tsx

```diff
@@ -0,0 +1,41 @@
+import cache from 'webext-storage-cache';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getRepo} from '../github-helpers';
+import * as api from '../github-helpers/api';
+
+const getWorkflowsCount = cache.function(async (): Promise<number> => {
+	const {workflows} = await api.v3('actions/workflows');
```

Comment:
Can you use API v4? It's a smaller response

Reply:
That'd make the feature behind the API token requirement. I was hoping that this feature could also benefit users who do not apply API tokens. But I can change it if you feel strongly about it.

Reply:
Use V4